content/en/docs/release-oversight/improving-ci-signal.md

```diff
@@ -0,0 +1,125 @@
+We will make an exception to the feature freeze requirements to allow un-reverts(along w/ the necessary fix) to land post feature freeze, even though technically the unrevert is (re)merging a feature PR.  Alternatively, consider moving your team to the no-feature-freeze process!
+
+
+### Vocabulary
+We often refer to these concepts with an imprecise vernacular, let’s try to codify a couple to make what follows easier to understand.
+* Job -- example: [periodic-ci-openshift-release-master-ci-4.9-e2e-gcp-upgrade](https://testgrid.k8s.io/redhat-openshift-ocp-release-4.9-informing#periodic-ci-openshift-release-master-ci-4.9-e2e-gcp-upgrade)
+* Jobs are a description of what set of the environment a cluster is installed into and which Tests will run and how. Jobs have multiple JobRuns associated with them.
+ JobRun -- example [periodic-ci-openshift-release-master-ci-4.9-e2e-gcp-upgrade #1423633387024814080](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.9-e2e-gcp-upgrade/1423633387024814080)
+* JobRuns are each an instance of a Job.  They will have N-payloads associated with them (one to install, N to change versions to).  JobRuns will have multiple TestRuns associated with them.
+JobRuns pass when all of their TestRuns also succeed.
+
+* TestRun -- example [`[sig-node] pods should never transition back to pending` from [periodic-ci-openshift-release-master-ci-4.9-e2e-gcp-upgrade #1423633387024814080](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.9-e2e-gcp-upgrade/1423633387024814080)
+A TestRun refers to an exact instance of Test for a particular JobRun.  At the files-on-disk level, this is represented by a junit-\*\*\*.xml file which lists a particular testCase.  Keep in mind that a single JobRun can have multiple TestRuns for a single Test.
+
+* Test -- example `[sig-node] pods should never transition back to pending`
+A Test is the bit of code that checks if a particular JobRun is functioning correctly.  The same Test can be used by many jobs and the overall pass/fail rates can provide information about the different environments set up by other jobs.
+
+* Payload Blocking Job -- click an instance of 4.9-nightly.  About 4 jobs.
+A job that is run on prospective payloads.  If the JobRun does not succeed, the payload is not promoted.
+
+* Payload Informing Job -- click an instance of 4.9-nightly.  About 34 jobs.
+A job that is run on prospective payloads.  If the Job run does not succeed, nothing happens: the payload is promoted anyway.
+
+* Flake Rate
+The percentage of the time that a particular TestRun might fail on a given payload, but may succeed if run a second time on a different (or sometimes the same) cluster.  These are often blamed on environmental conditions that result in a cluster-state that impedes success: load, temporary network connectivity failure, other workloads prevent test pod scheduling, etc.
```

Comment:
Can we add races to this list?  Or just something about chaos/indeterminism?  I think a bunch of our failures are based on a bunch of events which usually happen in a particular order and then give the happy result, but which will occasionally lose a race and fail.  And while that's sometimes going to be due to one of the "environmental conditions" currently listed here, it can also be brittle code in either the product or the test suite making assumptions that don't always hold up.

Reply:
I think that makes sense.  I'm not going to focus on getting this PR merged with minor edits.  You're welcome to submit a PR with helpful terms we should all agree on. :)